packages/sources/jpegd/test/integration/fixtures.ts

```diff
@@ -1,27 +1,52 @@
 import nock from 'nock'
 
-export const mockPunksValueResponseSuccess = (): nock =>
-  nock('https://jpegapi.com')
+export const mockPunksValueAPIResponseSuccess = (): nock =>
+  nock('https://jpegapi.com:443', { encodedQueryParams: true })
     .get('/punks')
-    .query({
-      block: 14000000,
-      api_key: 'test-key',
-    })
+    .query({ block: 'latest', api_key: 'test-key' })
+    .reply(200, { success: true, block: 14304636, value: 71.71525868055556 }, [
+      'Date',
+      'Wed, 02 Mar 2022 08:10:51 GMT',
+      'Content-Type',
+      'application/json',
+      'Transfer-Encoding',
+      'chunked',
+      'Connection',
+      'close',
+      'Vary',
+      'Accept-Encoding',
+    ])
+
+export const mockPunksValueResponseSuccess = (): nock =>
```

Comment:
This mock should be removed. Shouldn't mock what we're trying to test 😅

Reply:
Good catch

Reply:
@0xJPEG  Sorry, he meant that the`localhost:8080` mock should be removed

Reply:
Sorry, fixed